app/src/main/resources/io/apicurio/registry/storage/impl/sql/h2.ddl

```diff
@@ -40,6 +44,11 @@ CREATE HASH INDEX IDX_versions_5 ON versions(createdBy);
 CREATE INDEX IDX_versions_6 ON versions(createdOn);
 CREATE HASH INDEX IDX_versions_7 ON versions(contentId);
 
+CREATE TABLE artifact_version_branches (groupId VARCHAR(512) NOT NULL, artifactId VARCHAR(512) NOT NULL, branch VARCHAR(256) NOT NULL, branchOrder INT NOT NULL, version VARCHAR(256) NOT NULL);
+ALTER TABLE artifact_version_branches ADD PRIMARY KEY (groupId, artifactId, branch, branchOrder);
```

Comment:
Surprised to see `branchOrder` as part of the primary key.  Why?

Reply:
Oh I see - there is no actual "branches" table - this is actually a mapping table.  This table contains all of the mappings from version to branch, but there's no actual listing of just the branches.